modules/schedule/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "schedule",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "later": "latest",
```

Comment:
I'd prefer if these are the "safe" selectors. An easy way to get that would be to delete them from this and re-populate it via `npm install --save $package`. `latest` is scary.